modules/ROOT/pages/syntax-quick-reference.adoc

```diff
@@ -0,0 +1,1292 @@
+====
+include::text:example$text.adoc[tag=b-para]
+====
+
+.Literal paragraph
+[#ex-literal]
+----
+include::verbatim:example$literal.adoc[tag=qr-para]
+----
+
+.View result of <<ex-literal>>
+[%collapsible.result]
+====
+include::verbatim:example$literal.adoc[tag=qr-para]
+====
+
+.Lead paragraph
+[#ex-lead]
+----
+include::text:example$text.adoc[tag=qr-lead]
+----
+
+.View result of <<ex-lead>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=qr-lead]
+====
+
+The default Asciidoctor stylesheet automatically styles the first paragraph of the preamble as a xref:blocks:preamble-and-lead.adoc[lead paragraph] if no role is specified on that paragraph.
+
+== Formatted text
+
+.Bold, italic, and monospace
+[#ex-format]
+----
+include::text:example$text.adoc[tag=b-bold-italic-mono]
+----
+
+.View result of <<ex-format>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=b-bold-italic-mono]
+====
+
+.Highlight, underline, and strikethrough
+[#ex-lines]
+----
+include::text:example$text.adoc[tag=qr-all]
+----
+
+.View result of <<ex-lines>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=qr-all]
+====
+
+.Superscript and subscript
+[#ex-sub-sup]
+----
+include::text:example$text.adoc[tag=b-sub-sup]
+----
+
+.View result of <<ex-sub-sup>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=b-sub-sup]
+====
+
+.Smart quotes and apostrophes
+[#ex-curved]
+----
+include::text:example$text.adoc[tag=b-c-quote]
+----
+
+.View result of <<ex-curved>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=b-c-quote]
+====
+
+== Document header
+
+A xref:document:header.adoc[document header] is optional.
+The header may not contain blank lines and must be separated from the content by at least one blank line.
+
+.Title
+----
+include::document:example$title.adoc[tag=qr-title]
+----
+
+.Title and author line
+----
+include::document:example$header.adoc[tag=qr-author]
+----
+
+.Title, author line, and revision line
+----
+include::document:example$header.adoc[tag=qr-rev]
+----
+
+IMPORTANT: You cannot have a xref:document:revision-line.adoc[revision line] without an xref:document:author-line.adoc[author line].
+
+.Document header with attributes
+----
+include::document:example$header.adoc[tag=qr-attributes]
+----
+
+[#section-titles]
+== Section titles
+
+When the document type is `article` (the default), the document can only have one level-0 section title (`=`).
+This level-0 section title (i.e., the document title) must be in the document header.
+
+.Article doctype section levels
+[#ex-article]
+----
+include::sections:example$section.adoc[tag=base]
+----
+
+.View result of <<ex-article>>
+[%collapsible.result]
+====
+include::sections:example$section.adoc[tag=b-base]
+====
+
+The `book` document type can have additional level-0 section titles, which are interpreted as xref:sections:parts.adoc[parts].
+The presense of at least one part implicitly makes the document a multi-part book.
```

Comment:
Should be presence no? or maybe both presense and presence are valid?

Reply:
You are correct. I was fighting with the spell checker.